packages/auth/graphql/resolvers/User.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
 
   accessToken: (user, { scope }, { user: me }) => {
     if (Roles.userIsMeOrInRoles(user, me, DEFAULT_USER_ACCESS_ROLES)) {
-      return AccessToken.generateForUser(user, scope)
+      return AccessToken.issueForUser(me, user, scope)
```

Comment:
either remove the surrounding auth check or use `generateForUser`. Otherwise the double role checking makes this hard to understand the final restrictions.